monitor/metrics/checker.go

```diff
@@ -19,23 +19,33 @@ var ErrAlertChannelFull = errors.New("alert channel is full")
 // Checker provides utilities to find expired metrics
 // for a given peerset and send alerts if it proceeds to do so.
 type Checker struct {
-	alertCh chan *api.Alert
-	metrics *Store
+	alertCh   chan *api.Alert
+	metrics   *Store
+	threshold float64
 }
 
 // NewChecker creates a Checker using the given
 // MetricsStore.
-func NewChecker(metrics *Store) *Checker {
+func NewChecker(metrics *Store, threshold float64) *Checker {
```

Comment:
Maybe describe what is the threshold value in the comment above (and what are acceptable values).

Reply:
This is done.